Source/JavaScriptCore/runtime/JSCConfig.cpp

```diff
@@ -25,13 +25,18 @@
 
 #include "config.h"
 #include "JSCConfig.h"
+#include <wtf/WTFConfig.h>
 
 namespace JSC {
 
 #if !ENABLE(UNIFIED_AND_FREEZABLE_CONFIG_RECORD)
 Config g_jscConfig;
 #endif
 
+#if OS(WINDOWS) && ENABLE(WEBASSEMBLY)
+WTF::Config g_wtfConfigForLLInt = g_wtfConfig;
```

Comment:
This fails check-webkit-style for using an underscore in the identifier names, but I'm following the convention of `g_jscConfig`.

Reply:
I am not so sure this is right. Why is this needed?

Reply:
This is due to a limitation in offlineasm, which can't resolve the g_wtfConfig label from the DLL for WTF. By resolving the import on the C++ side, the g_wtfConfigForLLInt label is in the JSC DLL and can be resolved.

https://bugs.webkit.org/show_bug.cgi?id=175104

Reply:
I am still not sure this is correct? Shouldn't it be a reference then? Here, it seems like we copy the config. Anyway, since it is Windows-only, I won't block landing the patch on this.